### PR TITLE
docs (nodes): refactor doc strings for efficientdet node

### PR DESCRIPTION
**Pull request by chngzyk**

### CODE REVIEW

File: peekingduck/pipeline/nodes/model/efficientdet.py [hunk 1]
```diff
@@ -23,11 +23,42 @@
...
+        |bboxes|
+
+        |bbox_labels|
+
+        |bbox_scores|
+
+    Configs:
+        model_type (:obj:`int`): **{"0", "1", "2", "3", "4"}, default="0"**
+
+            Defines the compound coefficient for EfficientDet.
```
Comment (sidney-tio) [inline peekingduck/pipeline/nodes/model/efficientdet.py:47]: is it possible to give a short explanation of the compound coefficient? or at least what adjusting this config means for the user (slow, but more accurate model?)

File: peekingduck/pipeline/nodes/model/efficientdet.py [hunk 2]
```diff
@@ -23,11 +23,42 @@
...
+
+        detect_ids (:obj:`List`): **default = [0] **
+
+            List of object class ids to be detected.
+
+    References:
+
+    EfficientDet: Scalable and Efficient Object Detection: https://arxiv.org/abs/1911.09070
+
+    Code adapted from https://github.com/xuannianz/EfficientDet
```
Comment (sidney-tio) [inline peekingduck/pipeline/nodes/model/efficientdet.py:61]: indent this

File: peekingduck/pipeline/nodes/model/efficientdet.py (hunk 1 above)
Comment (chngzyk) [reply peekingduck/pipeline/nodes/model/efficientdet.py:50]: Noted. Added explanation on the compound coefficient.

**Comment by sidney-tio** [comment]
thanks! the addition is clear and concise

**Pull request merged**